region/brinstar/red.json

```diff
@@ -1283,21 +1287,22 @@
                   "requires": [
                     "canMidairShinespark",
                     "h_canXMode",
+                    "h_XModeSpikeHit",
                     {"canShineCharge": {
                       "usedTiles": 33,
                       "openEnd": 2
                     }},
                     {"shinespark": {
                       "frames": 146,
                       "excessFrames": 6
-                    }},
-                    {"spikeHits": 1}
+                    }}
                   ],
                   "note": [
                     "Jump from the entry runway to bounce onto spikes for XMode.",
                     "Store a spark in XMode and use that to shinepark across the room.",
                     "Bumping a solid tile before activating XMode will remove dash state, preventing shinecharging."
-                  ]
+                  ],
+                  "devNote": "Omitting leniency spikeHits, since the firefleas provide an opportunity to farm after a failed attempt."
```

Comment:
Confused here, since h_XModeSpikeHit has leniency built in

Reply:
Fixed, thanks!